packages/input-option/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@leafygreen-ui/input-option",
+  "version": "0.0.0",
```

Comment:
Shouldn't this be `0.9.0` so that It can be bumped to `1.0.0`? I think thats why `Polymorphic` did not initially get bumped in the version PR.

Reply:
I think as long as it's marked in the changeset as a major change, it should bump to 1.0.0 regardless